beta/src/components/MDX/Sandpack/Themes.tsx

```diff
@@ -10,7 +10,7 @@ export const CustomTheme = {
     defaultText: 'inherit',
     inactiveText: 'inherit',
     activeBackground: 'inherit',
-    defaultBackground: 'inherit',
+    defaultBackground: 'var(--sp-colors-bg-default)',
```

Comment:
this doesn't look good to me that this one alone is different. can you override it in `sandpack.css` instead like the rest?

Reply:
the reason i did it this way is because the styles specified here unfortunately don't get applied on the SSR pass. so we need to apply them "earlier"

Reply:
Done :heavy_check_mark: